docs/release-notes/8.4.0-1.md

```diff
@@ -0,0 +1,73 @@
+# Percona XtraDB Cluster 8.4.0-1 (2024-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release Highlights
+
+Percona XtraDB Cluster is based on Percona Server for MySQL. Find a complete list of improvements and bug fixes in the [Percona Server for MySQL 8.4.0-1 (2024-08-28) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.0-1.html).
+
+## Improvements
+
+* [PXC-4428](https://perconadev.atlassian.net/browse/PXC-4428): In asynchronous replicas, using `sql_log_bin=0` disrupted the Global Transaction Identifier (GTID) consistency, causing issues with replication and tracking transactions across the system.
```

Comment:
Something is wrong here. This one is about garbd. Listed below in 'bugs' section as well.